testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestHiveAndDeltaLakeCompatibility.java

```diff
@@ -1,90 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.tests.product.deltalake;
-
-import io.trino.tempto.BeforeMethodWithContext;
-import io.trino.tempto.ProductTest;
-import io.trino.tests.product.hudi.TestHudiHiveViewsCompatibility;
-import io.trino.tests.product.iceberg.TestIcebergHiveViewsCompatibility;
-import org.testng.annotations.Test;
-
-import static io.trino.tempto.assertions.QueryAssert.Row.row;
-import static io.trino.testing.TestingNames.randomNameSuffix;
-import static io.trino.tests.product.TestGroups.DELTA_LAKE_OSS;
-import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
-import static io.trino.tests.product.utils.QueryExecutors.onTrino;
-import static java.lang.String.format;
-import static java.util.Objects.requireNonNull;
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-
-/**
- * @see TestIcebergHiveViewsCompatibility
- * @see TestHudiHiveViewsCompatibility
- */
-public class TestHiveAndDeltaLakeCompatibility
-        extends ProductTest
-{
-    private String bucketName;
-
-    @BeforeMethodWithContext
-    public void setUp()
-    {
-        bucketName = requireNonNull(System.getenv("S3_BUCKET"), "Environment variable not set: S3_BUCKET");
-    }
-
-    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
-    public void testInformationSchemaColumnsOnPresenceOfHiveView()
```

Comment:
is `testInformationSchemaColumnsOnPresenceOfHiveView` already migrated?

Reply:
Yes, it's covered by the following test in my understanding:
https://github.com/trinodb/trino/blob/88e4a5cb435cee7884b72729f099d9c17c0e6e1d/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeSharedMetastoreViewsTest.java#L135-L154

Reply:
Is selecting from `information_schema.tables` equivalent to selecting from `information_schema.columns`?